recipe/meta.yaml

```diff
@@ -10,11 +10,13 @@ source:
   sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
 
 build:
-  number: 5
+  number: 6
   skip: true  # [win and py<37]
+  script_env:
+    - STAN_BACKEND=CMDSTANPY
   script: "{{ PYTHON }} -m pip install . -vv"
   binary_has_prefix_files:
-    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.pkl
+    - {{ SP_DIR|replace("\\", "/") }}/stan_model/prophet_model.bin
```

Comment:
Looking at the artifacts, I see that under Windows, the model file is `5,252,723` bytes, while under Linux it is `1,639,680`. I assume they are not compatible cross-platform? Is there any way that we could make a cross-platform model? (I am wondering if there is any hope for going noarch.)

Reply:
Not any more than it is possible to have a no-arch build of any compiled executable. They’re using different compilers and libcs, syscalls etc.

I think it’s possible to delete the precompiled header, I’ll try that locally